pages/linux/qm-guest-exec-status.md

```diff
@@ -0,0 +1,8 @@
+# qm guest exec-status
+
+> Get the status of the given pid started by the guest-agent on QEMU/KVM Virtual Machine Manager.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get status of a given pid:
```

Comment:
```suggestion
- Pring the status of a specific PID:
```

Reply:
can you update Pring to Print seems like a typo